frontend/src/app/classes/user.ts

```diff
@@ -0,0 +1,22 @@
+export class User {
+    public id: number
+    public name: string
+    public emailId: string
+    public username: string
+    public password: string
```

Comment:
add one more field for user type, as in backend there I have added user type filed to distinguish user

Reply:
I will, It's only the last version of the project and I will start working on it